src/cmd_handlers.c

```diff
@@ -1011,6 +1016,44 @@ goto_cmd(const cmd_info_t *cmd_info)
 	return 0;
 }
 
+static int
+regedit_cmd(const cmd_info_t *cmd_info)
+{
+	if (cmd_info->argc > 1)
+		return CMDS_ERR_TRAILING_CHARS;
+
+	const int regname = cmd_info->argc > 0 ? cmd_info->argv[0][0] : DEFAULT_REG_NAME;
+	if (regname == BLACKHOLE_REG_NAME)
+	{
+		ui_sb_err("Cannot modify blackhole register.");
+		return 1;
+	}
+
+	reg_t *reg = regs_find(tolower(regname));
+	if (NULL == reg || reg->nfiles == 0)
+	{
+		ui_sb_err("Nothing to modify in the empty register.");
+		return 1;
+	}
+	const int fname_len = PATH_MAX + 1;
+	char *tmp_fname = malloc((fname_len) * sizeof(char));
+	generate_tmp_file_name("vifm.regedit", tmp_fname, fname_len);
+	write_file_of_lines(tmp_fname, reg->files, reg->nfiles);
+	const int edit_result = vim_edit_files(1, &tmp_fname);
```

Comment:
Both functions return errors, which should be handled.  Temporary file should be removed on error or after reading it (no need to handle failure to remove it).

Reply:
I don't fully understand how I should handle these errors. For now this issues just raise error messages.

Reply:
See e689d14.

Reply:
That's fine. By handling I meant that they shouldn't be ignored and should trigger cleanup and error message.